voronoi_game.py

```diff
@@ -103,29 +107,15 @@ def __get_player_move(self):
     data = client_response.split()
     return int(data[0]), int(data[1])
 
+  def compute_pull(self, row, col):
+    return np.reciprocal(np.square(self.row_numbers - row) + np.square(self.col_numbers - col) + 0.1e-30)
```

Comment:
If we specifically handle the cell row, col in that case we won't be able to use optimizations of np.argmax used on line https://github.com/guyu96/Gravitational_Voronoi/pull/2/files#diff-307566ab5a58cec0e4fcc7bb6a2caaabR115

Reply:
Ahh so with this implementation we count the stones as well (which is also fine, they are equivalent by anything but constants).

What if we just made the addition + `0.1e-30` then the reciprocal will still be "infinity" or so, and would still work?

Reply:
In that case, it'll be same. For numbers even close to 1, e-30 doesn't make any difference.

Reply:
So, in my opinion take either of the routes you're comfortable with, It won't make any difference.

Reply:
Hmm yeah, maybe I should just give it up as in practice it won't really make a difference, the theoretical imperfection just annoys me :P.

Reply:
Okay, (didn't refresh before your previous message)

will probably just make that minor change just because I find it "neater"

Reply:
Have to tend to a few other things but will also do a few tests soon to make sure

Reply:
And then we should be able to merge it in :).

Thanks for the help!

Reply:
Hahaha! well, remember when we used to do sin(theta) = theta when theta < epsilon? I hated that with all my core!